lib/codeintel/upload/upload.go

```diff
@@ -189,13 +189,13 @@ func uploadMultipartIndexInit(ctx context.Context, httpClient Client, opts Uploa
 	return id, err
 }
 
-// uploadMultipartIndexParts uploads the contents available via each of the given reader
-// factories to a Sourcegraph instance as part of the same multipart upload as indiciated
+// uploadMultipartIndexParts uploads the contents available via each of the given reader(s)
```

Comment:
```suggestion
// uploadMultipartIndexParts uploads the contents available via each of the given readers
```

:laff: the parens were for emphasis